demos/common/python/openvino/model_zoo/model_api/models/model.py

```diff
@@ -84,10 +107,45 @@ def available_wrappers(cls):
 
     @classmethod
     def parameters(cls):
+        '''Defines the description and type of configurable data parameters for the wrapper.
+
+        See `types.py` to find available types of the data parameter. For each parameter
+        the type, default value and description must be provided.
+
+        The example of possible data parameter:
+            'confidence_threshold': NumericalValue(
+                default_value=0.5, description="Threshold value for detection box confidence"
+            )
+
+        The method must be implemented in each specific inherited wrapper.
```

Comment:
Currently, even if wrapper does not have configurable parameters, it defines `parameters()` method (`Deblurring` wrapper is the example). 
Actually it is not necessary, because the method can be inherited. So, this strong requirement helps to keep the wrappers aligned and directly define a wrapper empty of config parameters (unlikely, but someone might forget about `parameters()`)

I think the statement `"The method must..."` fits the current state of wrappers. If we refuse from definition of empty `parameters()` (and remove it from `Deblurring`, etc.), the phrase is needed to be rewritten as well

Reply:
Okey